app/assets/stylesheets/custom/search.scss

```diff
@@ -1,3 +1,16 @@
+/* override due to nesting govuk-width-container result in no visible margin */
+@media (min-width: 1020px) {
+  .moj-primary-navigation__container {
+    margin: 0 10px !important;
+  }
+}
+
+@media (min-width: 40.0625em) {
```

Comment:
You can use multiple media queries for a block

```css
@media (min-width: 1020px),
       (min-width: 16rem) // same as the em value you have
```

Reply:
Oh cool, I didn't know that.

Reply:
Yeah since it's the same rule, works the same as multiple selectors (since they are just selectors really)